test/liquid_handle_dispense_test.py

```diff
@@ -14,16 +18,303 @@ def __init__(self):
         self.name = "protein_buffer"
 
 
+class TestDispenseMethod(DispenseMethod):
+    def __init__(self):
+        super(TestDispenseMethod, self).__init__()
+        self._liquid = ProteinBuffer()
+
+
+def dummy_96():
+    return Container(
+        None,
+        ContainerType(
+            name="dummy",
+            well_count=96,
+            well_depth_mm=None,
+            well_volume_ul=Unit(200, "microliter"),
+            well_coating=None,
+            sterile=False,
+            is_tube=False,
+            cover_types=["universal"],
+            seal_types=None,
+            capabilities=["cover"],
+            shortname="dummy",
+            col_count=12,
+            dead_volume_ul=Unit(15, "microliter"),
+            safe_min_volume_ul=Unit(30, "microliter"),
+        ),
```

Comment:
why does ContainerType need to be passed here?

Reply:
Don't quite remember why I put it there, will remove if possible